packages/core/src/components/toggletip/toggletip.tsx

```diff
@@ -75,29 +118,140 @@ export class Toggletip {
     return;
   };
 
+  private positionTip(
+    referenceElement: HTMLElement,
+    tipRef: HTMLElement,
+    tipArrowRef: HTMLElement,
+    placement: Placement,
+    strategy: "absolute" | "fixed",
```

Comment:
Ik kan mij niet herinneren dat de originele functie `strategy` nodig had. Kan je toelichten waarom deze nodig is? Verder zie ik geen `popover` attributen, heb je die bewust niet gebruikt?